tests/Aspire.Hosting.Tests/Aspire.Hosting.Tests.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Aspire.Hosting.AppHost\Aspire.Hosting.AppHost.csproj" IsAspireProjectResource="false" />
+    <ProjectReference Include="..\..\src\Aspire.Hosting.Azure.AppContainers\Aspire.Hosting.Azure.AppContainers.csproj" IsAspireProjectResource="false" />
```

Comment:
We've been trying to keep Azure specific tests out of the Aspire.Hosting.Tests. If/when we want to push the Azure specific libraries to an Azure owned repo, we would need to do work to split these new tests out.

Reply:
These new tests should be moved to

https://github.com/dotnet/aspire/blob/b87cab2d9a8bb06aa0c8e5795a11d4aa2789c9ec/tests/Aspire.Hosting.Azure.Tests/AzureSchemaTests.cs#L9-L27

Reply:
The JSON schema is a bit of a monolith... This is a reflection of our product at the moment.

Reply:
The JSON schema can be a monolith - it doesn't mean our tests need to be.